pkgs/games/openttd/default.nix

```
@@ -123,27 +116,22 @@ stdenv.mkDerivation rec {
     "--without-liblzo2"
   ];
 
+  postPatch = ''
+    substituteInPlace src/music/fluidsynth.cpp \
+      --replace "/usr/share/soundfonts/default.sf2" "${soundfont-fluid}/share/soundfonts/FluidR3_GM2-2.sf2"
+  '';
+
   postInstall = ''
     ${lib.optionalString withOpenGFX ''
       cp ${opengfx}/*.tar $out/share/games/openttd/baseset
     ''}
```

```suggestion
  postInstall = lib.optionalString withOpenGFX ''
    cp ${opengfx}/*.tar $out/share/games/openttd/baseset
  '' + ''
```
please inline the conditionals like this